src/field.h

```diff
@@ -64,10 +64,13 @@ static int secp256k1_fe_is_zero(const secp256k1_fe *a);
 /** Check the "oddness" of a field element. Requires the input to be normalized. */
 static int secp256k1_fe_is_odd(const secp256k1_fe *a);
 
-/** Compare two field elements. Requires magnitude-1 inputs. */
+/** Compare two field elements. Requires both the inputs to have magnitude = 1.
+ *  i.e. a->magnitude = 1 and b->magnitude = 1 */
```

Comment:
Just a nit but why not just
```suggestion
/** Compare two field elements. Requires both the inputs to have magnitude 1. */
```
(Hm, in fact it's `<= 1` even but that should go away when  #1001 is solved). 

I don't think it's necessary to phrase it twice (and `x->magnitude` is only there #ifdef VERIFY -- but ok people would understands what is meant. ;)) 

Same below.

Reply:
Sure, I will remove this additional comment. I wanted to be more explicit, but I forgot that `x->magnitude` is used only in the VERIFY build.